src/directConnect.ts

```diff
@@ -43,26 +43,30 @@ export function openDirectConnectionForm(): void {
     // logger.debug("creating connection from form data:", body);
 
     let kafkaConfig: KafkaClusterConfig | undefined = undefined;
-    if (body["bootstrap_servers"]) {
-      kafkaConfig = {
-        bootstrap_servers: body["bootstrap_servers"],
-      };
+    if (body["clusterConfig"]) {
+      kafkaConfig = { ...body["clusterConfig"] };
     }
 
     let schemaRegistryConfig: SchemaRegistryConfig | undefined = undefined;
-    if (body["uri"]) {
-      schemaRegistryConfig = {
-        uri: body["uri"],
-      };
+    if (body["schemaConfig"]) {
+      schemaRegistryConfig = { ...body["schemaConfig"] };
     }
 
     const manager = DirectConnectionManager.getInstance();
-    return await manager.createConnection(
+    const result = await manager.createConnection(
       kafkaConfig,
       schemaRegistryConfig,
       body["name"],
       body["platform"],
     );
+    if (result.success) {
+      await window.showInformationMessage(`🎉 New Connection Created`, {
+        modal: true,
+        detail: `View and interact with ${body["name"]} in the Resources sidebar`,
```

Comment:
Should we default the name to something if the user doesn't set a name? The manager uses `New Connection` if nothing was passed, but we could set something generic like `your new connection` here instead.
<img width="521" alt="image" src="https://github.com/user-attachments/assets/5b0e84df-5f8e-4972-bb5e-cc4aaaa24953">

Reply:
ha good catch... I'll add something there as a placeholder